### PR TITLE
[OneExplorer] Add ArtifactLocator unit tests

### PR DESCRIPTION
**Pull request by dayo09**
This commit adds ArtifactLocator unit tests.

ONE-vscode-DCO-1.0-Signed-off-by: dayo09 <dayoung.lee@samsung.com>

----

Waiting for https://github.com/Samsung/ONE-vscode/pull/1062
Waiting for https://github.com/Samsung/ONE-vscode/pull/1074
Waiting for #1079

### CODE REVIEW

**Comment by hyunsik-yoon** [comment]
@dayo09 
I asked @llFreetimell about the ration of positive tests and negative tests and our guideline is 50%:50%. T.T
Could you add more negative tests?

**Comment by dayo09** [comment]
@hyunsik-yoon Sure :-D Can I add them on the next pr?

**Comment by hyunsik-yoon** [comment]
@dayo09 Of course, you can add any time. Thanks.

File: src/Tests/OneExplorer/ArtifactLocator.test.ts [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+ * limitations under the License.
+ */
+
+import {assert} from 'chai';
+import * as fs from 'fs';
+import * as vscode from 'vscode';
+import {Artifact, ArtifactAttr, Locator, LocatorRunner} from '../../OneExplorer/ArtifactLocator';
+
+// TODO Extract as test fixture
+class TestBuilder {
```
Comment (llFreetimell) [inline src/Tests/OneExplorer/ArtifactLocator.test.ts:23]: I think this class is not used for this test file... is there something I missed?

Comment (dayo09) [reply src/Tests/OneExplorer/ArtifactLocator.test.ts:23]: I will remove it. ;-D

**Comment by llFreetimell** [comment]
Format checker fail..!

**Comment by dayo09** [comment]
I added more negative tcs and added #1079 as a prerequisite of this PR.

**Comment by hyunsik-yoon** [comment]
Please ping us when you remove `NO_MERGE`. 😸

**Comment by dayo09** [comment]
@hyunsik-yoon @llFreetimell ping! :_D

**Review by llFreetimell — APPROVED** [review]

**Pull request merged**